From 47fff1d4304109c4c8e712b6291bc96c50acc1f3 Mon Sep 17 00:00:00 2001
From: Maciek <matt@mrek.co.uk>
Date: Thu, 20 Oct 2016 15:02:42 +0100
Subject: [PATCH] add authenticate config to .gitignore

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index e69de29..497c570 100644
--- a/.gitignore
+++ b/.gitignore
@@ -0,0 +1 @@
+simple-authenticate/config.php
\ No newline at end of file
-- 
2.7.4 (Apple Git-66)

